server/controllers/contests.js

```diff
@@ -5,113 +5,113 @@ const { userCheck } = require('../helpers/userCheck');
 
 //POST - create contest - auth
 exports.createContest = async (req, res, next) => {
-  const { title, description, prize, deadline, contestPics } = req.body;
-
-  //problem: possible to have unlimited contests and identical contests by the same user
-  //potential solution: store total contests in array as user property in DB
-
-  try {
-    //const user = await User.findOne({ _id: '5f61089819261d0d5680307f' });
-    const user = await User.findOne({ _id: req.user.userId });
-
-    const contest = new Contest({
-      title,
-      description,
-      prize,
-      deadline,
-      contestPics,
-      user,
-    });
-
-    await contest.save();
-    res.status(201).json(contest);
-  } catch (error) {
-    console.error(error.message);
-    res.status(500).json({ msg: 'Server error - 500' });
-  }
+	const { title, description, prize, deadline, contestPics } = req.body;
+
+	//problem: possible to have unlimited contests and identical contests by the same user
+	//potential solution: store total contests in array as user property in DB
+
+	try {
+		//const user = await User.findOne({ _id: '5f61089819261d0d5680307f' });
+		const user = await User.findOne({ _id: req.user.userId });
+
+		const contest = new Contest({
+			title,
+			description,
+			prize,
+			deadline,
+			contestPics,
+			user,
+		});
+
+		await contest.save();
+		res.status(201).json(contest);
+	} catch (error) {
+		console.error(error.message);
+		res.status(500).json({ msg: "Server error - 500" });
+	}
 };
 
 //GET - get contest by Id
 exports.getContestById = async (req, res, next) => {
-  try {
-    const contest = await Contest.findOne({ _id: req.params.id }).populate(
-      'user'
-    );
-
-    if (!contest) {
-      return res.status(404).json({ msg: 'This contest ID does not exist' });
-    }
-    res.status(200).json(contest);
-  } catch (error) {
-    console.error(error.message);
-    res.status(500).json({ msg: 'Server error - 500' });
-  }
+	try {
+		const contest = await Contest.findOne({ _id: req.params.id }).populate(
+			"user"
+		);
+
+		if (!contest) {
+			return res.status(404).json({ msg: "This contest ID does not exist" });
+		}
+		res.status(200).json(contest);
+	} catch (error) {
+		console.error(error.message);
+		res.status(500).json({ msg: "Server error - 500" });
+	}
 };
 
 //GET - get all contests
 exports.getAllContests = async (req, res, next) => {
   try {
     const contests = await Contest.find().populate('user');
 
-    if (!contests) {
-      return res.status(404).json({ msg: 'There are no contests' });
-    }
-    res.status(200).json(contests);
-  } catch (error) {
-    console.error(error.message);
-    res.status(500).json({ msg: 'Server error - 500' });
-  }
+		if (!contests) {
+			return res.status(404).json({ msg: "There are no contests" });
```

Comment:
looks like your prettier is linting some files - lets make sure you and @UVStudio 's linters are the same configs